docs/user/dashboard/drilldowns.asciidoc

```diff
@@ -1,52 +1,246 @@
 [[drilldowns]]
-== Use drilldowns for dashboard actions
+== Create custom dashboard actions
 
-Drilldowns, also known as custom actions, allow you to configure a
-workflow for analyzing and troubleshooting your data.
-For example, using a drilldown, you can navigate from one dashboard to another,
-taking the current time range, filters, and other parameters with you,
-so the context remains the same. You can continue your analysis from a new perspective.
+Custom dashboard actions, also known as drilldowns, allow you to create
+workflows for analyzing and troubleshooting your data. Drilldowns apply only to the panel that you created the drilldown from, and are not shared across all of the panels. Each panel can have multiple drilldowns. 
 
-[role="screenshot"]
-image::images/drilldown_on_piechart.gif[Drilldown on pie chart that navigates to another dashboard]
+{kib} supports three types drilldowns.
 
-Drilldowns are specific to the dashboard panel for which you create them—they are not shared across panels. A panel can have multiple drilldowns.
+[cols="50, 50"]
+|===
 
-[float]
-[[actions]]
-=== Drilldown actions
+a| *Dashboard drilldown*
 
-Drilldowns are user-configurable {kib} actions that are stored with the dashboard metadata.
-Kibana provides the following types of actions:
+| image:images/drilldown_on_piechart.gif[Drilldown on pie chart that navigates to another dashboard]
 
-[cols="2"]
-|===
+Navigate from one dashboard to another,
+taking the current time range, filters, and other parameters with you,
+so the context remains the same, then continue your analysis from a new perspective.
+For example, you might have a dashboard that shows the overall status of multiple data centers.
+You can create a drilldown that navigates from the overall status dashboard to a dashboard
+that shows a single data center or server.
+
+a| *URL drilldown*
 
-a| <<dashboard-drilldown, Dashboard drilldown>>
+| image:images/url_drilldown_go_to_github.gif[Drilldown on pie chart that navigates to Github]
 
-| Navigate to a dashboard.
+beta[] URL drilldowns allow you to navigate from a dashboard to an internal or external URL.
+The destination URL can be dynamic, depending on the dashboard context or user interaction with a panel.
+For example, you might have a dashboard that shows data from a Github repository.
+You can create a drilldown that navigates from the dashboard to Github.
 
-a| <<url-drilldown, URL drilldown>>
+Some panels support multiple user interactions, also known as triggers, that you can create for a URL drilldown. 
+The list of supported variables in the URL template depends on the trigger you choose.
+For example, pie chart triggers occur automatically when you click a slice, but area chart triggers occur when you 
+click a data point or select a range. You can create multiple URL drilldowns per panel and attach them to different triggers. 
+For more information about triggers, refer to <<url_templating-language, URL templating language>>.
```

Comment:
There was separate section explaining "triggers" (https://www.elastic.co/guide/en/kibana/master/url-drilldown.html#trigger-picker) and looks like the link doesn't explain that concept.